src/main/java/com/sports/server/auth/details/MemberDetails.java

```diff
@@ -0,0 +1,57 @@
+package com.sports.server.auth.details;
+
+import com.sports.server.command.member.domain.Member;
+import java.util.ArrayList;
+import java.util.Collection;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+@RequiredArgsConstructor
+public class MemberDetails implements UserDetails {
+
+    private final Member member;
+
+    public Member getMember() {
+        return member;
+    }
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        Collection<GrantedAuthority> authorities = new ArrayList<GrantedAuthority>();
+        authorities.add(new SimpleGrantedAuthority("user"));
```

Comment:
이건 한번 얘기 해봐야할거 같긴 한데 이전에 회의에서 우리가 추후에 사용자 로그인 (매니저 말고 그냥 사용자)를 추가할 수도 있다고 했던 기억이 있어.

사실 위와 같은 경우라면 Member가 권한을 들고 있는 형태로 바뀌거나 아니면 지금처럼 isManager를 확인해서 권한을 쥐어줘야 한다고 생각해.

따라서 위와 같은 경우에서는
```suggestion
        authorities.add(new SimpleGrantedAuthority("MANAGER"));
```

로 해야하지 않을까?

그리고 보통은 나는 Security 구현할때 권한을 아래와 같이 Enum Type으로 만들어 쓰는데 그러면 대문자로,, 권한을 주거든

```java
public enum AuthorityType {
    MANAGER,
    USER
}
```
그래서 대문자로 해야하지 않을까? 라는 생각도 있어

Reply:
아직은 사용자 로그인을 구현하는 것에 대해 자세한 논의가 이뤄지지 않아서 AuthorityType 과 같이 권한과 관련된 ENUM 은 추가하지 않고 user 에서 MANAGER 로만 수정해서 반영했어!